t/debug/hook.t

```diff
@@ -120,10 +125,35 @@ hook_test:                      # module and function list, name: hook_test
 GET /hello
 --- more_headers
 Host: foo.com
+X-APISIX-Dynamic-Debug: true
 --- response_body
 hello world
---- no_error_log
-[error]
 --- error_log
 filter(): call require("apisix.plugin").filter() args:{
 filter(): call require("apisix.plugin").filter() return:{
+
+
+
+=== TEST 5: missing hook_conf
+--- debug_config
+basic:
+  enable: true
+http_filter:
+  enable: true         # enable or disable this feature
+  enable_header_name: X-APISIX-Dynamic-Debug # the header name of dynamic enable
+
+hook_test:                      # module and function list, name: hook_test
+    apisix.plugin:              # required module name
+    - filter                    # function name
+
+#END
+--- request
+GET /hello
+--- more_headers
+Host: foo.com
+X-APISIX-Dynamic-Debug: true
+--- response_body
+hello world
+--- error_log
+read_debug_yaml(): failed to validate debug config property "hook_conf" is required
+--- wait: 3
```

Comment:
I think `wait: 2` is enough as we load debug yaml per second: https://github.com/apache/apisix/blob/f2326af32233442289cf9783f1022bfc5e532ca3/apisix/debug.lua#L311

Reply:
Anyway, wait for one more seconds is acceptable